### PR TITLE
My Orders View

### PR DESCRIPTION
**Pull request by CrhistianH**
This code implements "My Orders" view. It is fully working with Redux and loading the orders information from the back-end.
In order to complete this task, the following components were created as well:
- Order Card
- Order Info Modal

Also, I have added a minor adjustment to the "Privacy" and "Terms" buttons at the login screen. Now they are more button look-alike instead of being displayed as text.

### CODE REVIEW

**Comment by CrhistianH** [comment]
> Needs style refinement, some ugly things I found:
> image
> image
> something weird with margins and the borders, all within 'medium screen range', but there are different results with different widths within 'medium screen range'
> image
> Modal isn't centered
> image
> backdrop is small
> 
> The status indicator is too small

@pablorich I have fixed all of the issues you mentioned before. Please take another look at it.
I had to do some little modifications to the Status Label component that @AnaSandoval-Intersys created for us. I would appreciate that she let us know her feedback about the new changes on her component too.

File: src/views/MyOrders/my-orders.jsx [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+                        orders[i].billing_address.state + "), " + orders[i].billing_address.country + ". " +
+                        orders[i].billing_address.zip_code
+                    }
+                    shippedTo={
+                        orders[i].shipping_address.first_name + " " + orders[i].shipping_address.last_name + ". " +
+                        orders[i].shipping_address.address + ". " + orders[i].shipping_address.city + " (" +
+                        orders[i].shipping_address.state + "), " + orders[i].shipping_address.country + ". " +
+                        orders[i].shipping_address.zip_code
+                    }
+                    cost={orders[i].total}
```
Comment (pablorich) [inline src/views/MyOrders/my-orders.jsx]: add decimal dot

File: src/views/MyOrders/my-orders.jsx [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+
+        let products = [];
+
+        if (orders[i].status !== "Pending") {
+
+            for (let n = 0; n < orders[i].products.length; n++) {
+                products.push(
+                    orders[i].products[n].quantity + "x " + orders[i].products[n].name
+                )
+            }
```
Comment (pablorich) [inline src/views/MyOrders/my-orders.jsx]: You're sending an array of products but when you finally display all the products they're in the same line

**Comment by alf-null** [comment]
Well done!!

**Pull request closed without merging**